rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1335,6 +1335,8 @@ def test_upgrade_db_36_to_37(user_data_dir):  # pylint: disable=unused-argument
     ]
     assert all(x in old_history_events for x in kraken_events)
     assert all(x in old_history_events for x in custom_events)
+    old_ens_mappings = cursor.execute('SELECT * FROM ens_mappings').fetchall()
```

Comment:
Question: I don't see you editing the test DB. Did it already have ENS mappings? I guess yes.

Reply:
Yes, it did